webapp/src/main/resources/application.yml

```diff
@@ -89,6 +89,10 @@ permissionservice:
   endpoint: ${rdw.permission.service}
 
 app:
+  reporting:
+    webapp:
+      url: /
```

Comment:
Should this be http://localhost/ or some such?  I guess i don't know what works for local devs but, for my own edification, this is expected to be a fully URL, correct?  e.g. https://rdw-prod-reporting/?